composables/api/User/UserControllerDto.d.ts

```diff
@@ -382,3 +384,89 @@ export type RequestSendVerificationCodeResponseDto = {
 	/** 附加的文本消息 */
 	message?: string;
 };
+
+/**
+ * 邀请码类型
+ */
+type InvitationCode = {
+	/** 生成邀请码的用户 - 非空 */
+	creatorUid: number;
+	/** 邀请码 - 非空 */
+	invitationCode: string;
+	/** 生成邀请码的时间 - 非空 */
+	generationDateTime: number;
+	/** 邀请码被标记为等待使用中 - 非空 */
+	isPadding: boolean;
+	/** 邀请码被标记为无法使用 - 非空 */
+	disabled: boolean;
+	/** 使用这个邀请码的用户 */
+	assignee?: number;
+	/** 邀请码被使用的时间 */
+	usedDateTime?: number;
+};
+
+/**
+ * 生成邀请码的请求响应
+ */
+export type CreateInvitationCodeResponseDto = {
+	/** 执行结果，程序执行成功，返回 true，程序执行失败，返回 false */
+	success: boolean;
+	/** 是否超出邀请码生成期限 */
+	isOutOfTimeLimit: boolean;
```

Comment:
```suggestion
	isExpired: boolean;
```
expired：过期的
我认为这里用过期更为直观

Reply:
这里是“超过时间限制”
或许用 isPassTimeLimit 好一些，我觉得不应该用 isExpired

Reply:
change to `isCoolingDown`